providers/capvcd/standard/test_data/cluster_values.yaml

```diff
@@ -3,38 +3,33 @@ controlPlane:
   catalog: giantswarm
   replicas: 3
   sizingPolicy: m1.large
-  template: ubuntu-2004-kube-v1.22.9
+  template: ubuntu-2004-kube-v1.24.10
 connectivity:
-  bastion:
-    enabled: true
   network:
     loadBalancers:
-      vipSubnet: "178.170.32.1/24"
+      vipSubnet: "10.205.9.254/24"
   proxy:
-    enabled: false
+    enabled: true
 nodePools:
   worker:
     class: default
     replicas: 2
 providerSpecific:
-  org: giantswarm
-  ovdc: vDC 73640
-  site: https://vmware.ikoula.com
-  ovdcNetwork: capvcd-192.168.52.0
-  cloudProviderInterface:
-    enableVirtualServiceSharedIP: false
-    oneArm:
-      enabled: true
+  org: GIANT_SWARM
+  ovdc: Org-GIANT-SWARM
+  site: "https://cd.neoedge.cloud"
+  ovdcNetwork: GS-ISOLATED
   nodeClasses:
     default:
       catalog: giantswarm
       sizingPolicy: m1.large
-      template: ubuntu-2004-kube-v1.22.9
+      template: ubuntu-2004-kube-v1.24.10
   userContext:
     secretRef:
       secretName: vcd-credentials
+  vmNamingTemplate: neoedge{{.machine.Name | sha256sum | trunc 4}}
```

Comment:
This file is handled as a Go template file, you'll need to escape these if you want to have them passed through to the actual Helm values.

On a side note, there's a LOT of values in here. WAY more than I'd expect a custom to need to provide. Are they all required? Where possible we should be relying on the chart default values otherwise this is going to be quite brittle as things get updated.

Reply:
yes these are all required, there are no defaults between customer vsphere environments so we need customers to provide them. unfortunately this set of values is the bare minimum required.